local_planner/src/nodes/local_planner.cpp

```diff
@@ -123,11 +123,13 @@ void LocalPlanner::runPlanner() {
   // visualization of FOV in RViz
   initGridCells(FOV_cells_);
   geometry_msgs::Point p;
+  PolarPoint p_pol = {};
```

Comment:
If you don't want to put data in it you don't need the ` = {}`. However, you aren't accessing it outside the loop, so just declare it below inside the loop when you set it.